src/reportWebVitals.js

```diff
@@ -1,15 +0,0 @@
-const reportWebVitals = (onPerfEntry) => {
```

Comment:
- [ ] You have done well removing this boilerplate file, However, you need to remove the ones in the `public` directory.

  ![jmhy](https://github.com/meryemsanem/Math-Magicians/assets/36542986/8269f7a8-6f2c-4d1d-8700-5f6422f4bcdc)